src/test/kotlin/no/nav/syfo/identhendelse/IdenthendelseServiceSpek.kt

```diff
@@ -115,6 +115,47 @@ object IdenthendelseServiceSpek : Spek({
                     val updatedMotedeltakerArbeidstaker = database.getPersonOversiktStatusList(newIdent.value)
                     updatedMotedeltakerArbeidstaker.size shouldBeEqualTo 0
                 }
+
+                it("Skal overskrive veilederident når ny ident allerede finnes i databasen") {
+                    val kafkaIdenthendelseDTO = generateKafkaIdenthendelseDTO(hasOldPersonident = true)
+                    val newIdent = kafkaIdenthendelseDTO.getActivePersonident()!!
+                    val oldIdent = kafkaIdenthendelseDTO.getInactivePersonidenter().first()
+
+                    val veilederIdent = "Z990099"
+                    val personOversiktStatusWithOldIdent = PersonOversiktStatus(
+                        fnr = oldIdent.value,
+                        veilederIdent = veilederIdent,
+                    )
+                    val personOversiktStatusWithNewIdent = PersonOversiktStatus(
+                        fnr = newIdent.value,
+                        veilederIdent = null,
+                    )
+                    database.connection.use { connection ->
+                        connection.createPersonOversiktStatus(
+                            commit = true,
+                            personOversiktStatus = personOversiktStatusWithOldIdent,
+                        )
+                    }
+                    database.connection.use { connection ->
+                        connection.createPersonOversiktStatus(
+                            commit = true,
+                            personOversiktStatus = personOversiktStatusWithNewIdent,
+                        )
+                    }
+
+                    runBlocking {
+                        identhendelseService.handleIdenthendelse(kafkaIdenthendelseDTO)
+                    }
+
+                    // Check that person with new personident is updated with veilederident
```

Comment:
Det er vel ikke noe som står i denne kommentaren som ikke står i koden? Kan den fjernes?

Reply:
Jo, men jeg ser på variabelnavnene at jeg har copy+pastet litt for mye her 🙃 

Men poenget er at den nye identen har `veilederIdent = null`, men så kopierer vi over verdien fra den gamle identen til den nye, før den slettes.